recipes/qhull/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,23 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
-add_executable(test_package test_package.c)
-target_link_libraries(test_package ${CONAN_LIBS})
+find_package(Qhull REQUIRED CONFIG)
+
+if(QHULL_REENTRANT)
```

Comment:
DO you know if the upstream export any definition, it could be useful.

Reply:
No custom definition added in config file. Here is the custom input config file: https://github.com/qhull/qhull/blob/master/build/config.cmake.in